Library/Homebrew/cask/lib/hbc/cli/internal_stanza.rb

```diff
@@ -68,10 +65,17 @@ def run
             next
           end
 
-          if artifact_name && !value.key?(artifact_name)
-            opoo "no such stanza '#{artifact_name}' on Cask '#{cask}'" unless quiet?
-            puts ""
-            next
+          if value.nil? || (value.respond_to?(:to_a) && value.to_a.empty?) ||
+             (value.respond_to?(:to_s) && value.to_s == "{}") ||
```

Comment:
This does not apply to any case, because a hash responds to `to_a`, and will be handled by the check above.

Reply:
As I said above, seems like I am lacking a bit of knowledge regarding Ruby Sets and Arrays, but I will get there ...